frontend/src/components/WaitingForPlayer.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const WaitingForPlayer: React.FunctionComponent = () => {
```

Comment:
This component is for when there is no enemy in royale mode (in between rounds)?

Reply:
Nope. Let's stay battle starts with 8 people. When 1 dies, 1 stays without a pair, and so, until the next player dies, someone will have to wait each round.

Reply:
Or, when 1 player disconnects, same scenario.